pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -248,17 +253,21 @@ func (gb *GraphBuilder) startMonitors() {
 	// that they don't get unexpected events on their work queues.
 	<-gb.informersStarted
 
+	gb.sharedInformers.Start(gb.stopCh)
 	monitors := gb.monitors
```

Comment:
what is this line for?

Reply:
If I'm remembering our discussing, this line is needed to start informers that have been stopped (and are newly added) but is redundant for informers that are already running, but there isn't a clean way to avoid this redundancy. Is that right?